src/main/resources/openapi/audiorequests.yaml

```diff
@@ -393,6 +417,30 @@ paths:
                 type: "AUDIO_101"
                 title: "The requested data cannot be located"
                 status: 500
+    post:
+      tags:
+        - Audio Requests
+      summary: Users can request audio for a hearing and date/time periods
+      operationId: addAudioRequestPlayback
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AddAudioResponse'
+        '400':
+          description: 'invalid input, object invalid'
+        '401':
+          description: Unauthorised Error
+        '409':
+          description: audio request item already exists
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AudioRequestDetails'
```

Comment:
Implementation Notes:
If existing audio-request endpoints are removed in future then we should remove AudioRequestType from AudioRequestDetails (or at least make not-required). For now, the request_body still needs request_type although this will be set by the new endpoints to playback or download.

front end ticket dmp-2521 expects request body to be unchanged. i will raise this so that when old endpoint is removed, these issues are sorted